.eslintrc.cjs

```diff
@@ -1,8 +1,17 @@
 module.exports = {
-  extends: 'react-app',
+  extends: [
+    'react-app',
+    'plugin:@typescript-eslint/recommended-type-checked',
+    'plugin:@typescript-eslint/stylistic-type-checked'
+  ],
 
   parser: '@typescript-eslint/parser',
 
+  parserOptions: {
+    project: true,
```

Comment:
[`project: true` is already in typescript-eslint@v5](https://typescript-eslint.io/linting/typed-linting). It's just a nice way to enable `parserOptions.project` 😄